.vscode/extensions.json

```diff
@@ -1,10 +1,11 @@
 {
     "recommendations": [
-      "ms-vscode.csharp",
+      "ms-dotnettools.csharp",
       "ms-vscode.cpptools",
       "ms-vscode.mono-debug",
-      "wghats.vscode-nxunit-test-adapter",
       "visualstudioexptteam.vscodeintellicode",
       "streetsidesoftware.code-spell-checker",
+      "wghats.vscode-nxunit-test-adapter",
```

Comment:
Why reorder this?  Is order important?

Reply:
Order was not important. I just decided to put all the unit test adapters at the bottom.